loadgen/loadgen.cc

```diff
@@ -827,6 +827,14 @@ struct PerformanceSummary {
   PercentileEntry latency_percentiles[6] = {{.50}, {.90}, {.95},
                                             {.97}, {.99}, {.999}};
 
+#if defined(_WIN32) || defined(WIN32) || defined(_WIN64) || defined(WIN64)
+  // MSVC complains if there is no explicit constructor.
+  // (target_latency_percentile above depends on construction with settings)
```

Comment:
This dependency isn't necessary since ProcessLatency gets it from target_latency_percentile after construction.

Adding an explicit constructor with this signature makes the std::moves not move, causing linters to complain and slightly reducing performance of the metrics calculations.

Can we instead set the target_latency_percentile to something like .9999, so:
1) it'll be obvious if it's not set later on in ProcessLatency.
2) the dependency on target_latency_percentile is removed.

Reply:
Yes, its needed. msvc folks actually fixed it but it will take some time before real users have updated:
https://developercommunity.visualstudio.com/content/problem/717625/fatal-error-c1001-mlperf-c-module-doesnt-compile-o.html
We could try to set target_latency_percentile but I'm not sure if there are other implications.

Reply:
lgtm. Not sure how to handle this with regards to freeze though.

Reply:
Since it is ifdef windows it should not break anything.
I think the rules state that one need to build the frozen loadgen ... if this doesn't make it into the freeze we are excluding windows submissions.